apps/condo/domains/notification/schema/Message.js

```diff
@@ -119,7 +116,7 @@ const Message = new GQLListSchema('Message', {
             },
         ],
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), historical()],
+    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
```

Comment:
Maybe to move it after `historical`, like in other places?

Reply:
No. It's important to put it before. In other cases, this field will not be included in history. historical should be the last